farming/logics/traits/data.rs

```diff
@@ -47,7 +47,7 @@ pub struct Data {
     pub lp_tokens: Vec<AccountId>,
 
     /// Address of each `rewarder` contract in MasterChef.
-    pub rewarders: Vec<AccountId>,
+    pub rewarders: Vec<Option<AccountId>>,
```

Comment:
I don't think there is a reason to store None values.
Best solution would be to store Hashmap<(id: u32, rewarder: AccountId), ()>
So you can easily check if an rewarder exist
Or  Hashmap<id: u32, rewarder: AccountId> and only store where is is a rewarder (it will be None if id is not existing)
Id can be the pool id that will be stored (so pool_id +1)

Reply:
Agree, I'll work on that using Hashmap as you suggested. For now, let me deploy it to Shibuya, and I'll reflect this change later.

Reply:
Fixed as of https://github.com/AstarNetwork/wasm-showcase-dapps/pull/12/commits/8e2dd3a2f5b9eed533084fbdf97a5484cd1b05bd